### PR TITLE
Feature/build secrets

### LINKED ISSUE #101: Apply secure xcconfig build configuration technique
Follow pattern of https://github.com/rakutentech/ios-remote-config/pull/34 and implement secure config using gitignored xcconfig file.

### PR DESCRIPTION
**Pull request by Climbatize**
# Description
Prevents API keys, urls or certificate passwords to be pushed into sources by error
This PR is a step to local server security improvement
Closes #101 

## Links
https://pages.ghe.rakuten-it.com/ssed/tech-blog/posts/ios-secure-xcconfig/

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data before every commit, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

File: configure-secrets.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+echo "//" >> $SECRETS_FILE
+echo "// Add new secrets configuration in ./configure-secrets.sh" >> $SECRETS_FILE
+echo "//" >> $SECRETS_FILE
+echo "// In order to have the // in https://, we need to split it with an empty" >> $SECRETS_FILE
+echo "// variable substitution via \$() e.g. ROOT_URL = https:/\$()/www.endpoint.com" >> $SECRETS_FILE
+
+# Set secrets from environment variables
+echo "RMA_API_ENDPOINT = $RMA_API_ENDPOINT_SECRET" >> $SECRETS_FILE
+echo "RAS_PROJECT_SUBSCRIPTION_KEY = $RAS_PROJECT_SUBSCRIPTION_KEY_SECRET" >> $SECRETS_FILE
+echo "RAS_APPLICATION_IDENTIFIER = $RAS_APPLICATION_IDENTIFIER_SECRET" >> $SECRETS_FILE
```
Comment (donnie-jp) [inline configure-secrets.sh:39]: needs newline?

File: Config/Config-debug.xcconfig [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#include "../Pods/Target Support Files/Pods-MiniApp_Example/Pods-MiniApp_Example.debug.xcconfig"
```
Comment (donnie-jp) [inline Config/Config-debug.xcconfig:1]: The xcconfig files are specifically for the example app so should they be in the `Example/` folder?

Comment (Climbatize) [reply Config/Config-debug.xcconfig:1]: that makes sense

File: configure-secrets.sh (hunk 1 above)
Comment (Climbatize) [reply configure-secrets.sh:39]: Yes, App Studio needs a subscription key and a project key for Mini App

Comment (donnie-jp) [reply configure-secrets.sh:39]: I meant GitHub is complaining cos there isn't a newline character at the end of the file.

Comment (Climbatize) [reply configure-secrets.sh]: Yeah this is a `C` legacy of git, so you know your EOL occurs before the EOF. Most editors force it, but not Sublime Text :) I did not notice it, thanks

**Pull request merged**